data/replaced_tags.txt

```diff
@@ -59,6 +59,8 @@ ford=yes : highway=ford
 barrier=jersey_barrier : barrier=block
 barrier=sliding_gate : barrier=gate
 
+# Icons.mapcss does not allow using something=no, so we change "no" to "not"
+drinking_water=no : drinking_water=not
```

Comment:
As far as I understand/remember, it should be enough to 
```
man_made|water_tap|drinking_water_no;[man_made=water_tap][drinking_water=not],[man_made=water_tap][drinking_water=no];;;;1604;
```
It'll match all `drinking_water=no`, the replaced_tags.txt change is not needed.

Alternatively (but IMHO its worse than above) it could be:
```
  replaced:
drinking_water=no : drinking_water=not

  mapping:
man_made|water_tap|drinking_water_no;[man_made=water_tap][drinking_water=not];;;;1604;
```

The reason is that the `replaced_tags.txt` is a preprocessor.  So past it all `drinking_water=no` are converted to `drinking_water=not` already.

Reply:
Please add also `drinking_water:legal=no` (4.5k cases) support, e.g.
```
man_made|water_tap|drinking_water_no;[man_made=water_tap][drinking_water=not],[man_made=water_tap][drinking_water=no],[man_made=water_tap][drinking_water:legal=no];;;;1604;
```

And `natural=spring drinking_water=no` 6k cases.

(there are also 2k cases of `amenity=drinking_water drinking_water=no` and 2.6k `amenity=water_point drinking_water=no` which look like mistagging)

Reply:
@pastk Thank you so much for your help! Sometimes when dealing with the styles here, it's a bit of a mystery what's going to work and what's not 😂

Replaced tags is not necessary. Removed.

I've added non-legal drinking too.

Added natural-spring non-drinking too.